ggshield/scan/scanner.py

```diff
@@ -91,13 +116,37 @@ def has_iac_result(self) -> bool:
     def has_results(self) -> bool:
         return bool(self.results and self.results.results)
 
+    def _init_has_new_secrets(self) -> bool:
+        """
+        Returns true when at least one new secret is found in the results of this scan (subscans are NOT checked)
+        """
+        if self.results and self.results.results:
+            return not all(
+                policy_break.known_secret
+                for result in self.results.results
+                for policy_break in result.scan.policy_breaks
+            )
+        return False
+
+    def _init_known_new_secrets_count(self) -> Tuple[int, int]:
```

Comment:
nitpick: naming is surprising: I expect a `_init_something(self)` to define `self.something`, not to return it. I would replace the `_init` prefix with something like `_get`, `_compute`, or just remove it.